pkg/common/common.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/go-logr/logr"
 	kyvernov1 "github.com/kyverno/kyverno/api/kyverno/v1"
+	kyvernov1beta1 "github.com/kyverno/kyverno/api/kyverno/v1beta1"
```

Comment:
they might be deprecating this one. please check if there is similar function in v2beta1

Reply:
There is no constant.go file in v2beta1, so we won't be able to move this constant there.